.github/workflows/nightly-build.yml

```diff
@@ -0,0 +1,74 @@
+name: Nightly Build
+
+on:
+  workflow_dispatch:
+  pull_request:
+  schedule:
+    - cron: '0 10 * * *' # Run every day at 10AM UTC
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.21.x
+
+      - name: Run linting tests
+        id: update
+        run: |
+          make prepare
+          make lint
+          make fmt-check
+          make imports-check
+
+      - name: Run unit tests
+        run: |
+          make coverage-ci
+
+      - name: Build cross-platform binaries
+        run: |
+          make build-cli-cross-platform
+
+      - name: Run integration tests
+        env:
+          CI_ACCOUNT: ${{ secrets.CI_ACCOUNT }}
+          CI_SUBACCOUNT: ${{ secrets.CI_SUBACCOUNT }}
+          CI_API_KEY: ${{ secrets.CI_API_KEY }}
+          CI_API_SECRET: ${{ secrets.CI_API_SECRET }}
+          LW_INT_TEST_AWS_ACC: ${{ secrets.LW_INT_TEST_AWS_ACC }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y vim
```

Comment:
Why do we install vim?

Reply:
vim is needed for integrations test of inline editing LQL queries. There is a ticket for having a docker image for this: https://lacework.atlassian.net/browse/GROW-924